include/Partitioning/HDRF.hpp

```diff
@@ -184,7 +183,7 @@ namespace CXXGRAPH
                     cord_state.incrementMachineLoadVertices(machine_id);
                 }
             }
-            catch (std::bad_cast)
+            catch (std::bad_cast& e)
```

Comment:
Could the excpetion be passed as "const" ? Aesthetic change, slightly more secure.

Reply:
resolved in 2ad39b9c3a66666490c5a3ab3807cfbed78547f6